extensions/typescript-language-features/src/languageFeatures/diagnostics.ts

```diff
@@ -149,31 +150,68 @@ class DiagnosticSettings {
 	}
 }
 
-class DiagnosticsTelemetryManager {
-
-	private _diagnosticCodes: number[] = [];
+/*
+const diagnostics = this.getDiagnostics(document.uri);
+			const diagnoticCodes = diagnostics.reduce(function (result: number[], d: vscode.Diagnostic) {
+				const code = d.code;
+				if (typeof code === 'string' || typeof code === 'number') {
+					result.push(Number(code));
+				} else if (code !== undefined) {
+					result.push(Number(code.value));
+				}
+				return result;
+			}, []);
+			*/
```

Comment:
leftover comment

Reply:
That comment was not in the PR at the moment when a review was asked, I think the PR on your side may have been outdated